src/components/AccountDetails/index.tsx

```diff
@@ -217,7 +217,8 @@ interface AccountDetailsProps {
   ENSName?: string
   openOptions: () => void
 }
-
+const mockPendingTx = ['0x123123123', '0x1235asdasdasd']
```

Comment:
Why changes in `@src/.../AccountDetails` if you override it in `custom/` anyway

Reply:
ah sorry this should be removed!